invenio_jobs/administration/jobs.py

```diff
@@ -46,7 +46,7 @@ class JobsAdminMixin:
             "icon": "calendar",
         },
         "runs": {
-            "text": "Run now",
+            "text": "Configure and run",
```

Comment:
```suggestion
            "text": "Configure and run...",
```
... to indicate it opens a modal. It is already less scary to click :)